src/sdk/services/blockchain/contracts/IDContract.ts

```diff
@@ -333,8 +333,8 @@ class IDContract {
         } else if (account instanceof Name) {
             // use the account name directly
             data = await api.v1.chain.get_table_rows({
-                code: 'id.tonomy',
-                scope: 'id.tonomy',
+                code: 'id.tmy',
```

Comment:
lets create a constant with the contract name in this file for use everywhere here